src/swagger-specs/data-hygiene.yaml

```diff
@@ -0,0 +1,1485 @@
+    search:
+      name: search
+      in: query
+      schema:
+        type: string
+      description: >
+        Matches expirations where the specified string is an exact match for the expiration ID, or is contained in any of 
+        these fields: `author`, `display name`, `description`, `display name`, `dataset name`.
+      example: "John Q. Public"
+    status:
+      name: status
+      in: query
+      schema:
+        type: string
+      description: >
+        A comma-separated list of statuses. When included, the response matches dataset expirations whose current status is among 
+        those listed.
+      example: pending,canceled
+    updatedDate:
+      name: updatedDate
+      in: query
+      schema:
+        type: string
+      description: >
+        Like `createdDate` / `createdFromDate` / `createdToDate`, but matches against a dataset expiration’s update time instead 
+        of creation time. An expiration is considered updated on every edit, including when it is created, cancelled, or executed.
+      example: '2022-01-01'
+    updatedToDate:
+      name: updatedToDate
+      in: query
+      schema:
+        type: string
+      description: >
+        Like `createdDate` / `createdFromDate` / `createdToDate`, but matches against a dataset expiration’s update time instead 
+        of creation time. An expiration is considered updated on every edit, including when it is created, cancelled, or executed.
+      example: '2022-01-01'
+    updatedFromDate:
+      name: updatedFromDate
+      in: query
+      schema:
+        type: string
+      description: >
+        Like `createdDate` / `createdFromDate` / `createdToDate`, but matches against a dataset expiration’s update time instead 
+        of creation time. An expiration is considered updated on every edit, including when it is created, cancelled, or executed.
+      example: '2022-01-01'
+    productStatusDetails:
+      name: productStatusDetails
+      in: query
+      required: false
+      schema:
+        type: string
+      description: >
+        This parameter is only accepted when the expiration status is `executing`, or `completed`. Each element of the array contains 
+        a `productName` and a `productStatus`. The `productName` will be either `Data Management`,  `Identity Service`, or 
+        `Profile Service`. The `productStatus` will always be `success`.
+    requestId:
+      name: requestId
+      in: query
+      required: false
+      schema:
+        type: string
+      description: >
+        An optional workorder ID for a specific record delete request that you want to retrieve.
+
+  schemas:
+    ttlPutPayload:
+      type: object
+      properties:
+        expiry:
+          type: string
+          example: "2030-12-31T23:59:59Z"
+          description: >
+            The expiry date and time in ISO 8601 format. If the string has no explicit zone offset, UTC is 
+            assumed. This must be at least 24 hours in the future.
```

Comment:
I don't think saying "is assumed" is the best way to phrase it. I'd go with like "by default, this is UTC" or something like that.